src/corebluetooth/utils/mod.rs

```diff
@@ -16,11 +16,22 @@
 // This file may not be copied, modified, or distributed except
 // according to those terms.
 
-use objc2_foundation::NSUUID;
+use std::ffi::CStr;
+
+use objc2_foundation::{NSString, NSUUID};
 use uuid::Uuid;
 
 pub mod core_bluetooth;
 
 pub fn nsuuid_to_uuid(uuid: &NSUUID) -> Uuid {
     uuid.UUIDString().to_string().parse().unwrap()
 }
+
+pub fn nsstring_to_string(nsstring: *const NSString) -> Option<String> {
```

Comment:
This function should be marked as `unsafe`, as it can cause undefined behaviour if `nsstring` is not a valid pointer.